### ISSUE TITLE
I meet some error when run you script

### ISSUE THREAD

**Issue by sdrgroup** [j1z0/vim-config#1]
After run your script, I got following message. I don't know what's wrong with it. Could you help me solve it? Thank you.
line   65:
E319: Sorry, the command is not available in this version: py << EOF
line   66:
E492: Not an editor command: import os.path
line   67:
E492: Not an editor command: import sys
line   68:
E492: Not an editor command: import vim
line   69:
E15: Invalid expression: 'VIRTUA_ENV' in os.environ:
line  126:
E171: Missing :endif
Error: Required vim compiled with +python

My OS is ubuntu16.04

**Comment by j1z0** [comment]
This vimrc requires VIM compiled with python.

Seems like this thread would have some good info for you:
http://askubuntu.com/questions/764882/ubuntu-16-04-vim-without-python-support

**Comment by leelong2020** [comment]
#### I get the same " Error ", but  I can't execute the command "apt install vim-nox-py2" with Debian.

#### So what should I do?

**Comment by WonderM** [comment]
Hi.

I am a Linux noobs and am trying your script on Linux Mint 19 and these are the error messages I get:
Error detected while processing /home/XXXXX/.vim/vimrc:
line   66:
E319: Sorry, the command is not available in this version: py << EOF
line   67:
E492: Not an editor command: import os.path
line   68:
E492: Not an editor command: import sys
line   69:
E492: Not an editor command: import vim
line   70:
E15: Invalid expression: 'VIRTUA_ENV' in os.environ:
line  127:
E171: Missing :endif
Error: Required vim compiled with +python
I tried to follow this link (http://askubuntu.com/questions/764882/ubuntu-16-04-vim-without-python-support) and vim-nox-py2 is can not be located.

Have you tried your script on the latest Ubuntu or Linux Mint?
Where can I learn more about these errors?
Is it possible to help or update your script?

Thanks

**Comment by j1z0** [comment]
yeah basically you need vim compiled with python 2 support for that to work.. .if you vim has python 3 support you can change `py << EOF` to `py3 << EOF` but either way you need some sort of python support compiled with your VIM.

**Comment by bwakkie** [comment]
> yeah basically you need vim compiled with python 2 support for that to work.. .if you vim has python 3 support you can change `py << EOF` to `py3 << EOF` but either way you need some sort of python support compiled with your VIM.

thanks, that helped me :)
